Eastern-Research-Group/mywaterway
# Feature/20 nars updates

## Related Issues:
* https://jira.epa.gov/browse/HMW-20

## Main Changes:
* Brad found an instance of `(opens new browser tab)` that was not wrapped in `<small>` tags. I updated code so that all instances of `(opens new browser tab)` are wrapped in `<small>` tags.

## Steps To Test:
1. Navigate to http://localhost:3000/national
2. Verify all instances of `(opens new browser tab)` are using smaller font.
